packages/utils/regl-renderer/demo.html

```diff
@@ -137,6 +147,34 @@
   window.requestAnimationFrame(updateAndRender)
 }
 window.requestAnimationFrame(updateAndRender)
+
+// convert HTML events (mouse movement) to viewer changes
+let lastX = 0
+let lastY = 0
+
+const rotateSpeed = 0.002
+let rotateDelta = [0, 0]
+
+const enterHandler = (ev) => {
+  lastX = ev.pageX
+  lastY = ev.pageY
+
+  ev.preventDefault()
+}
+
+const moveHandler = (ev) => {
+  rotateDelta[0] = ev.pageX - lastX
```

Comment:
use += for delta so it does not skip stesp, otherwise, looks nice, This also a nice sample how to run render without most ! 😸 
```js
  rotateDelta[0] += ev.pageX - lastX
  rotateDelta[1] += lastY - ev.pageY
```